features/step_definitions/link_pension_webchat_to_pension_and_retirement_pages_steps.rb

```diff
@@ -1,18 +1,39 @@
-Given(/^I visit an article( not| NOT)? within the Pension and Retirement category in my "([^"]*)"$/) do |should_not, language|
-	article =
-    if should_not
-      {
-        'English' => '/en/articles/saving-money-for-christmas',
-        'Welsh' => '/cy/articles/sut-i-gynilo-ar-gyfer-y-nadolig'
-      }
-    else
-      {
-        'English' => '/en/articles/personal-pensions',
-        'Welsh' => '/cy/articles/pensiynau-personol'
-      }
-    end
+Given(/^I visit "([^"]*)" not within the Pension and Retirement category in my "([^"]*)"$/) do |entity, language|
+  links = {
+    'article' => {
+      'English' => '/en/articles/saving-money-for-christmas',
+      'Welsh' => '/cy/articles/sut-i-gynilo-ar-gyfer-y-nadolig'
+    },
+    'category' => {
+      'English' => '/en/categories/help-with-your-benefits',
+      'Welsh' => '/cy/categories/help-with-your-benefits'
+    },
+    'tool' => {
+      'English' => '/en/tools/budget-planner',
+      'Welsh' => '/cy/tools/cynllunydd-cyllideb/start'
+    }
+  }
 
-  visit(article[language])
+  visit(links[entity][language])
+end
+
```

Comment:
@gotbadger this is what I was thinking. Wdyt, please?

```ruby
Given(/^I visit "([^"]*)" not within the Pension and Retirement category in my "([^"]*)"$/) do |entity, language|
  visit(links_outside_pensions_and_retirement_category[entity][language])
end

Given(/^I visit "([^"]*)" within the Pension and Retirement category in my "([^"]*)"$/) do |entity, language|
  visit(links_within_pensions_and_retirement_category[entity][language])
end

private

  def links_outside_pensions_and_retirement_category
    {
       'article' => {
         'English' => '/en/articles/saving-money-for-christmas',
         'Welsh' => '/cy/articles/sut-i-gynilo-ar-gyfer-y-nadolig'
       },
       'category' => {
         'English' => '/en/categories/help-with-your-benefits',
         'Welsh' => '/cy/categories/help-with-your-benefits'
       },
       'tool' => {
         'English' => '/en/tools/budget-planner',
         'Welsh' => '/cy/tools/cynllunydd-cyllideb/start'
       }
    }
  end

  def links_within_pensions_and_retirement_category
    {
      'article' => {
        'English' => '/en/articles/personal-pensions',
       'Welsh' => '/cy/articles/pensiynau-personol'
      },
      'category' => {
        'English' => '/en/articles/options-for-using-your-pension-pot',
        'Welsh' => '/cy/articles/yr-opsiynau-ar-gyfer-defnyddioch-cronfa-bensiwn'
      },
      'tool' => {
        'English' => '/en/tools/pension-calculator',
        'Welsh' => '/cy/tools/cyfrifiannell-pensiwn/'
      }
    }
  end
```

Reply:
@Tr4pSt3R since the code is not in any class it feels weird to throw a private in here. Probably better if we just assign the hash to a constant since the values are fixed and freeze it. 

```Ruby
LINKS_IN_PENSIONS_AND_RETIREMENT = {
      'article' => {
        'English' => '/en/articles/personal-pensions',
       'Welsh' => '/cy/articles/pensiynau-personol'
      },
      'category' => {
        'English' => '/en/articles/options-for-using-your-pension-pot',
        'Welsh' => '/cy/articles/yr-opsiynau-ar-gyfer-defnyddioch-cronfa-bensiwn'
      },
      'tool' => {
        'English' => '/en/tools/pension-calculator',
        'Welsh' => '/cy/tools/cyfrifiannell-pensiwn/'
      }
    }.freeze
```

WDYT?

Reply:
_You is smart!_ Haha :) 
Yes that should be better.